code/obj/flock/structure/flock_structure_parent.dm

```diff
@@ -42,7 +42,7 @@ ABSTRACT_TYPE(/obj/flock_structure)
 	/// maximum allowed tick spacing for mult calculations due to lag
 	var/cap_tick_spacing = FLOCK_PROCESS_SCHEDULE_INTERVAL * 5
 
-/obj/flock_structure/New(var/atom/location, var/datum/flock/F=null)
+/obj/flock_structure/New(var/atom/location, var/datum/flock/F, atom/param)
```

Comment:
this is still called `param`

Reply:
Yeah I clarified above, was just kept this way to represent an arbitrary atom parameter reservation that can be used by Flock structures for whatever reason. Ghost buildings aren't really "converted from" a type. Going to leave as is unless it's an issue still